mdm/common/src/main/java/com/gentics/mesh/core/data/dao/PersistingNodeDao.java

```diff
@@ -2052,6 +2070,8 @@ default void addReferenceUpdates(HibNode updatedNode, BulkActionContext bac) {
 			});
 	}
 
+	Stream<? extends HibNodeFieldContainerEdge> findLanguageEdges(HibProject project, Collection<String> languageTags);
```

Comment:
This is only used to determine, whether any of the given languages is currently used in the project.
It is not necessary to return a Stream of all ContainerEdges here, maybe a map of language tag to boolean would be sufficient.
This way, we could find more efficient ways to determine the required information.